ignition/service/queue.py

```diff
@@ -47,18 +48,19 @@ def __init__(self):
         self.consumer_group_id = 'job_queue_consumer'
 
 class JobQueueCapability(Capability):
-    
+
     @interface
     def queue_job(self, job):
         pass
-    
+
     @interface
     def register_job_handler(self, job_type, handler_func):
         pass
 
 class MessagingJobQueueService(Service, JobQueueCapability):
 
     JOB_TYPE_KEY = 'job_type'
+    version = QUEUE_MESSAGE_VERSION
```

Comment:
Should this be in the constructor (__init__) so that it can be overridden? Does it need to be a property so that it can be overridden by properties (the same applies to the request queue and other message send capabilities too)?

Reply:
I had put it here as other constant class variable 'JOB_TYPE_KEY' was put there as well. But changed now to make it part of init in commit 6.